src/acpi-tables/src/madt.rs

```diff
@@ -0,0 +1,123 @@
+// Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::mem::size_of;
+
+use vm_memory::{Address, Bytes, GuestAddress, GuestMemory};
+use zerocopy::little_endian::U32;
+use zerocopy::AsBytes;
+
+use crate::{checksum, AcpiError, Result, Sdt, SdtHeader};
+
+const MADT_CPU_ENABLE_FLAG: u32 = 0;
+
+#[repr(packed)]
+#[derive(Copy, Clone, Debug, Default, AsBytes)]
+pub struct LocalAPIC {
+    _type: u8,
+    _length: u8,
+    _processor_uid: u8,
+    _apic_id: u8,
+    _flags: U32,
+}
+
+impl LocalAPIC {
+    pub fn new(cpu_id: u8) -> Self {
+        Self {
+            _type: 0,
+            _length: 8,
+            _processor_uid: cpu_id,
+            _apic_id: cpu_id,
+            _flags: U32::new(1u32 << MADT_CPU_ENABLE_FLAG),
```

Comment:
nit: no underscores on any of these

Reply:
I ignore the lint and everything is without an underscore now.